app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'sz.nedzynski@gmail.com'
```

Comment:
you can start receiving spam if it is a real email address - bots might pick it up from repo :D